docs/ml_copies.md

```diff
@@ -70,3 +70,70 @@ The Replacement Capability is the ratio between the accuracy of the copy model w
+
+
+### Model selection
+
+ * **Test data**
+ 
+If some labeled original test data is available, it is always more reliable to use that to compute the metrics and carry out the copy evaluation. Even if that original data is not labeled, it can always be labeled using the prediction of the original model and then be used to evaluate the copy. But this will at least ensure that the sampling is done following the original feature distribution.
+       
+If no original data is available, then a test synthetic dataset can be generated for evaluation of the copy. But be aware that, if this dataset is generated with the same sampler that was used to carry out the copy, the test will focus on the same regions where the copy has had more training. Hence, the fidelity or the performance of the copy will not be evaluated as rigorously on the outskirt decision boundary.
+
+ * **Sampling in evaluation**
+
+What does it mean if the copy has a low empirical fidelity error with synthetic data, but at the same time has a high fidelity error with test data? 
+
+The copy is often created without having a priori an idea of the distribution of the real data, this means that real data may be located mostly in a particular region while synthetic data will tend to be all over the place. This means that the original model which has been trained with real data will have a stronger emphasis on having certain boundaries right, while the copy model will be trained with data that enforces to have a similar boundary as the original mostly everywhere.
+
+When a copy is better than another in the synthetic data, but is worse in the test data, this normally means that the first copy resembles the original better overall but perhaps in the region where the real data are concentrated it doesn't have a particularly good performance. Therefore, it is important to try to have a copy with a good boundary everywhere (so that the region of interest is also included), or to try to determine the distribution of the real dataset, or the region of interest of the real data, when this is possible.
+
+
+### Sampling
+
+ * **Standardize the features**
+
+Certain samplers, such as the spherical sampler in this package, may not handle well a large difference in the absolute magnitude of the different features, or that they are not centered in the origin. Therefore, it is advisable to standardize the input data in case of doubt. However, it is unavoidable with this sampler, that classes in more centric regions will be better sampled that more peripherical ones.
+
+ * **Reduce the features space**
+ 
+Although ML classifier copies can be carried out without any access to the original data, at least some assumptions about the relevant range of the features is necessary. The more precise the assumptions that can be made, the more efficient the sampling will be, which is specially relevant in problems with a large number of dimensions.
```

Comment:
I would emphasize here that the data that we're generating through sampling needs to be accepted as input for prediction by the original model, so we need to something about the schema it expects, eg. data types, ranges, etc.

Reply:
Added.